packages/core-data/src/resolvers.js

```diff
@@ -25,15 +25,34 @@ import {
 import { getKindEntities } from './entities';
 import { apiFetch, resolveSelect } from './controls';
 import { ifNotResolved } from './utils';
+import getQueryParts from './queried-data/get-query-parts';
 
 /**
  * Requests authors from the REST API.
  */
-export function* getAuthors() {
+export function* getAuthors( query = {} ) {
+	const path = addQueryArgs( '/wp/v2/users/?who=authors&per_page=100', {
+		...query,
```

Comment:
👍 will fix

Reply:
removing this spread breaks searching - as we pass `{ search: [string] }` to `getAuthors`. we could remove if I set this here to `'search': query`, is that preferable?  this would allow only searching and not passing other query parameters.

Reply:
@gwwar Thanks for the feedback, I'll give that a try. I think the main issue is making sure the query arrives correctly because the API endpoint expects `{ search: "string" }`.

Reply:
Fixed in [`d97d175` (#23237)](https://github.com/WordPress/gutenberg/pull/23237/commits/d97d175d75b4ebfa5be6131d3023b5576e837ce1)